README.md

```diff
@@ -1 +1,41 @@
-# to-do-list
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# To-do List
+
+In this project we build a aplication called to-do list where you can create a list of thigs have to do, this aplication was built with plenty JavaScript
+
+## Built With
+
+- Major languages
+- HTML, CSS, Javascript
+- Webpack
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps:
+
+1. Go to the [repository page](https://github.com/rega1237/to-do-list).
+2. Press the "Code" button and copy the link.
+3. Clone it using git command `git clone <link>`.
+
```

Comment:
- [ ] Kindly provide instructions in your Readme on how to install dependencies of the project(`npm install`) as well as running it on the local server(`npm start`).